.github/workflows/release_notes_updated.yml

```diff
@@ -28,7 +30,9 @@ jobs:
         run: |
           if [[ $(expr match "${{ github.event.pull_request.head.ref }}" "release_v[0-9.]\+") -gt 0 ]]; then
             exit 0;
-          elif [[ $(expr match "${{ github.event.pull_request.head.ref }}" "dep-update-[a-zA-Z0-9]*") -gt 0 ]]; then
+          elif [[ $(expr match "${{ github.event.pull_request.head.ref }}" "latest-dep-update-[a-zA-Z0-9]*") -gt 0 ]]; then
```

Comment:
Updated the release notes check to skip latest and minimum dependency branches

Reply:
Good call
There's probably a fancy unix way to do this in one `if` check hahaha but I don't know what it is right away at least